### DISCUSSION TITLE
UI/UX Designer ENG/FARSI USA

### DISCUSSION THREAD

**Discussion by aggie-for** [WeAreMahsaAmini/Core#430] [Introductions]
Hello, UI/UX designer, based in Texas, born in Iran. I specialize in mobile apps. Able to help with graphics and any front end project as well. من زبان فارسی را هم می‌خوانم هم مینویسم.

**Comment by tcf-admin** [comment]
@aggie-for, 
check out forDem's introduction here: https://github.com/WeAreMahsaAmini/Core/discussions/495
Hope you find it interesting and we can have you next to us.

**Reply by aggie-for** [reply]
Thank you. I subscribed to it. It looks like they are only needing developers? Should I comment and ask if there is a need for UI/UX and front end work?

**Reply by tcf-admin** [reply]
Of course there is a need for UI/UX designer and frontend developers. Currently there are several projects where you could directly jump in and be a part of the production, including forDem, Iranliberationmovement.com and so on. Should I help you get involve quickly?
